### ISSUE TITLE
Possibility to use different compression format for resulting artifact

### ISSUE THREAD

**Issue by lbeuster** [excelsior-oss/excelsior-jet-maven-plugin#79]
Unfortunately the created zip artifact cannot be unpacked on busybox embedded linux because of the limitations there (unzip: zip flags 1 and 8 are not supported). So it would be nice to offer a configuration option to choose a different format (e.g. tar.gz) for the resulting output artifact.

**Comment by pjBooms** [comment]
Do you try Excelsior JET Embedded? What host OS are you using (Linux, Windows)?

**Comment by lbeuster** [comment]
Yes, we use a licensed Excelsior JET Embedded 15.3 on Linux.

**Comment by pjBooms** [comment]
I have just released 1.3.1 version of the plugin with "tar-gz" packaging type.  So if you add it to the configuration section of the plugin:

```xml
          <plugin>
              <groupId>com.excelsiorjet</groupId>
              <artifactId>excelsior-jet-maven-plugin</artifactId>
              <version>1.3.1</version>
              <configuration>
                  <mainClass></mainClass>
                  <packaging>tar-gz</packaging>
              </configuration>
          </plugin>
```

the plugin will create tar.gz file containing your jet-compiled application in the `target/jet` subdirectory of your Maven project.
Please let us know if it works for you.

**Comment by lbeuster** [comment]
Thanks, works 👍
